ledger-core/virtual/execute/delegatedtokenrequest_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package execute
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gojuno/minimock/v3"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/smsync"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/contract"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/node"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger"
+	messageSender "github.com/insolar/assured-ledger/ledger-core/network/messagesender"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/object"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils/slotdebugger"
+)
+
+func TestVirtual_CDelegatedCallRequest(t *testing.T) {
+	var (
+		mc       = minimock.NewController(t)
+		ctx      = inslogger.TestContext(t)
+		deadBeef = []byte{0xde, 0xad, 0xbe, 0xef}
```

Comment:
Isn't it better to use exported value from package? @sexyhamster

Reply:
hmm. Its better to create tokenService here and created expected token from arguments. Then you won't need deadBeef bytes

Reply:
That var private in test of other package. They do not have a general meaning and can be changed independently.

Reply:
as far as I can see, there is no need in these bytes

Reply:
done